gogen/gogen.go

```diff
@@ -1162,46 +1163,8 @@ func (t *{{ .ParentReceiver }}) To_{{ .Name }}(i interface{}) ({{ .Name }}, erro
 	]", i, i)
 }
 `)
-
-	// templateHelperFunctions specifies a set of functions that are supplied as
-	// helpers to the templates that are used within this file.
-	templateHelperFunctions = template.FuncMap{
-		// inc provides a means to add 1 to a number, and is used within templates
-		// to check whether the index of an element within a loop is the last one,
-		// such that special handling can be provided for it (e.g., not following
-		// it with a comma in a list of arguments).
-		"inc": func(i int) int {
-			return i + 1
-		},
-		"toUpper": strings.ToUpper,
-		"indentLines": func(s string) string {
-			var b bytes.Buffer
-			p := strings.Split(s, "\n")
-			b.WriteRune('\n')
-			for i, l := range p {
-				if l == "" {
-					continue
-				}
-				b.WriteString(fmt.Sprintf("  %s", l))
-				if i != len(p)-1 {
-					b.WriteRune('\n')
-				}
-			}
-			return b.String()
-		},
-		// stripAsteriskPrefix provides a template helper that removes an asterisk
-		// from the start of a string. It is used to remove "*" from the start of
-		// pointer types.
-		"stripAsteriskPrefix": func(s string) string { return strings.TrimPrefix(s, "*") },
-	}
 )
 
-// mustMakeTemplate generates a template.Template for a particular named source
```

Comment:
Since I think this induces a panic, I wonder whether we actually want to create it in a shared package -- I'm OK with doing so (since it's an internal package too), but it's essentially a small saving.

Reply:
Makes sense, I moved this back into gogen.